wiki/Modding/Bubble/vi.md

```diff
@@ -0,0 +1,55 @@
+---
+tags:
+ - bubble pop
+ - full bubble
+---
+
+# Bubble
+
+Cái **bubble** (![Bubble icon](/wiki/shared/icon/bubble.gif)) là một biểu tượng bài đăng trên diễn đàn được sử dụng bởi các thành viên của các nhóm modding sau đây tham gia vào [Quá trình xếp hạng beatmap](/wiki/Beatmap_ranking_procedure):
+
+- [Beatmap Appreciation Team](/wiki/People/Beatmap_Appreciation_Team) (*BAT*)
+- [Mapping Assistance Team](/wiki/People/Mapping_Assistance_Team) (*MAT*)
+- [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (*BN*)
+- [Quality Assurance Team](/wiki/People/Quality_Assurance_Team) (*QAT*)
+
+Là một bước tiếp theo của [Hệ thống ranking cũ](/wiki/Modding/Forum_modding) sau [Proto-bubble](/wiki/Modding/Proto-bubble), một bubble bình thường có nghĩa là một [beatmap](/wiki/beatmap) đã chuẩn bị được xếp hạng từ ý kiến của modder. Những beatmap được bubble sẽ được kiểm tra bởi một thành viên khác của BAT hoạc BN và được [xếp hạng](/wiki/beatmap/category#ranked) hay [chấp nhận](/wiki/beatmap/category#approved) miễn là chỉ có những thay đổi nhỏ cho beatmap là bắt buộc.
+ 
+Trong [quy trình xếp hạng beatmap](wwiki/Beatmap_Ranking_Procedure) hiện , cái tương đương với bubble là [đề cử](/wiki/beatmap_ranking_procedure#nominations) được cho bởi một [người đề cử beatmap](/wiki/people/beatmap_nominators).
+ 
+## Lịch sử
+ 
+*Xem thêm [Dòng thời gian mapping và modding](/wiki/History_of_osu!/Mapping_And_Modding_Timeline)*
+ 
+Bubble được giới thiệu bởi ::{ flag=AU }:: [peppy](/wiki/People/peppy) vào ngày 29 tháng 10 năm 2007, cho "những beatmap đang được xem xét cho lượt chơi đã được xếp hạng (đang chờ phản hồi xa hơn bởi mod)". Đặt một biểu tường cho thread của beatmap sang một cái bubble là một cách cho thành viên BAT để chỉ ra rằng beatmap có chất lượng tốt và đã theo [tiêu chí xếp hạng](/wiki/Ranking_Criteria)<!-- internal reference: https://osu.ppy.sh/community/forums/topics/619 -->.
+
+Vào ngày 3 tháng 10 năm 2010, Nhóm MAT đã nhận được quyền để sử dụng biểu tượng bubble, cùng với BAT!
+
+Sau [hệ thống thảo luận beatmap](/wiki/beatmap_discussion) đã được hoàn toàn thêm vào và đã hóa thành giao diện chính cho việc modding vào tháng 11 năm 2017, quản lý beatmap dựa trên diễn đàn đã bị loại bỏ dần, với bubble được thay bằng đề cử.
+
+## Cơ chế
+
+::: Infobox
+![](img/list-of-bubbles.png "Danh sách những beatmap được bubble, cùng với những bubble được nổ và những map được đánh dấu sao")
+:::
+
+Thread của những beatmap được bubble đã được liệt kê trên những trang đầu tiên của [Forum những beatmap đang chờ](https://osu.ppy.sh/community/forums/6), đã được sắp xếp bởi [sự ưu tiên sao](/wiki/modding/star_piority) của chúng và được hiệu lực theo vai những chuỗi forum.
+
+### Yêu cầu
+
+- Để được bubble, [sự ưu tiên sao](/wiki/modding/star-piority) của một beatmap phải có ít nhất là 8.
```

Comment:
```suggestion
- Để được bubble, [sự ưu tiên sao](/wiki/modding/Star_priority) của một beatmap phải có ít nhất là 8.
```
Just going to mention that I can't see any reason for `/wiki/modding/Star_priority` to be linked twice and also considering that they are used within the same subject matter.